p2p/monero-p2p/src/network_zones/clear.rs

```diff
@@ -16,6 +16,10 @@ use crate::{NetZoneAddress, NetworkZone};
 impl NetZoneAddress for SocketAddr {
     type BanID = IpAddr;
 
+    fn set_port(&mut self, port: u16) {
+        self.set_port(port)
+    }
```

Comment:
This seems infinitely recursive?

Reply:
hmm yeah, this does work though, Rust must resolve impls before trait impls 

I'll change this to `SocketAddr::set_port(self, port)`